cmake/configs/default.cmake

```diff
@@ -58,7 +58,7 @@ hunter_default_version(Comet VERSION 4.0.2)
 hunter_default_version(CppNetlib VERSION 0.10.1-hunter-3)
 hunter_default_version(CppNetlibUri VERSION 1.0.5-hunter)
 hunter_default_version(CsvParserCPlusPlus VERSION 1.0.1)
-hunter_default_version(Eigen VERSION 3.3.4-p1)
+hunter_default_version(Eigen VERSION 3.3.5-p0)
```

Comment:
this should be version `3.3.5`. The `-p0` extension indicates a hunter specific patch

Reply:
@NeroBurner fixed in 1685d34